test/src-tests/index-test.js

```diff
@@ -46,6 +60,30 @@ describe('find-parent/index', function() {
         assert.isUndefined(result);
       });
 
+      it('should return undefined when no parent matches and throwOnMiss is false', function() {
+        var opts = {
```

Comment:
Do you want to pull these up to the top and have a `optsThrowOnMiss` and a `optsDontThrowOnMiss` so they can be reused?